modules/core/src/v2/coins/eos.ts

```diff
@@ -1,19 +1,22 @@
 /**
  * @prettier
  */
-import { BaseCoin, BaseCoinTransactionExplanation } from '../baseCoin';
+import { BaseCoin, BaseCoinTransactionExplanation, BaseCoinInitiateRecoveryOptions } from '../baseCoin';
 import { NodeCallback } from '../types';
 import { BigNumber } from 'bignumber.js';
 import * as crypto from 'crypto';
-import * as utxoLib from 'bitgo-utxo-lib';
+import { HDNode } from 'bitgo-utxo-lib';
 import * as EosJs from 'eosjs';
 import * as ecc from 'eosjs-ecc';
 import * as url from 'url';
 import * as querystring from 'querystring';
 import * as _ from 'lodash';
-import { InvalidAddressError, UnexpectedAddressError } from '../../errors';
 import * as Bluebird from 'bluebird';
 const co = Bluebird.coroutine;
+import { InvalidAddressError, UnexpectedAddressError } from '../../errors';
+import * as config from '../../config';
+import * as sjcl from '../../vendor/sjcl.min.js';
+import * as common from '../../common';
```

Comment:
Please import `Environments` from `v2/environments.ts` instead of importing through common.